runtime/testy/studentCode.py

```diff
@@ -80,4 +80,5 @@ def hibikeSubscribeDevice_setup():
 
 def hibikeSubscribeDevice_main():
   Robot.hibikeSubscribeDevice(1, 2, [3, 4])
+  time.sleep(0.01)
```

Comment:
Not necessary, just had it to check whether the value was supposed to change after a device was subscribed and whether it was a timing issue that kept the correct value from being read.